fpx/src/api/models.rs

```diff
@@ -16,7 +16,7 @@ pub const FPX_WEBSOCKET_ID_HEADER: &str = "fpx-websocket-id";
 
 /// Messages that are send from the server to the client.
 #[derive(JsonSchema, Clone, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
Why have you added the `deny_unknown_fields` option? This might make it impossible to have any backwards compatible models (ie. adding a new optional field will be rejected by older clients, since this new field is "unknown")

Reply:
Unfortunately this is the only way to add `additionalProperties: false` to the JSON schema's. Without this the TS types generated are not really strict, and allow for any property: e.g:

```ts
type MyType = {
   [x: string]: any,
   actualPropertyThatIsInTheSchema: bool, // etc
};
```

So the validation function using ajv still checks for it properly, but any types we add to our own functions become pretty much useless...